qemu/tests/steal_time.py

```diff
@@ -0,0 +1,72 @@
+import logging
+import time
+import re
+
+from virttest import error_context
+from virttest import utils_test
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Steal time test:
+    1) Boot two guests and bind to one same cpu
+    2) Run stress inside both guests
+    3) Check steal time in top inside guests
+    4) Check steal time in /proc/stat in guest
+    5) Repeat step 4) after 60s, compare the steal time changed in two guests
+
+    :params test: QEMU test object.
+    :params params: Dictionary with the test parameters.
+    :params env: Dictionary with test environment.
+    """
+    def get_stat_val():
+        """
+        Get steal time value in /proc/stat
+        """
+        stat_val = []
+        for session in sessions:
+            val = session.cmd_output(stat_cmd).split()[8]
+            stat_val.append(int(val))
+        return stat_val
+
+    stress_args = params["stress_args"]
+    stress_tests = []
+    sessions = []
+    vms = env.get_all_vms()
+
+    error_context.context("Run stress in both guests", logging.info)
+    for vm in vms:
```

Comment:
This design can not ensure the two vms run at the same time ?

Reply:
As long as they are both running and stressed before check steal time, then it's fine.